spawn/spawn.go

```diff
@@ -160,3 +180,106 @@ func CreateHost(so Options) (*host.Host, error) {
 	}
 	return intentHost, errors.WithStack(err)
 }
+
+func SetHostRDPPassword(ctx context.Context, host *host.Host, password string) error {
```

Comment:
The terminate host handler below passes in settings directly, rather than using the global accessor here, should we try to be consistent here?

Reply:
I'm now using the spawn.SetHostRDPPassword function in the route Execute method directly (i.e. without a data connector method), so I didn't want to put the settings object as a parameter. 

If i add a DB connector method, i'm adding the first context to DBHostConnector.
If I leave it as is, we're using the Evergreen env settings behind the scenes.